pkg/unittest/unittest.go

```diff
@@ -130,7 +130,7 @@ func (r *Runner) Run() error {
 
 			outputFile := filepath.Join(r.OutputDir, file.Name())
 			if r.Update {
-				err := ioutil.WriteFile(outputFile, testResult, 0644)
+				err := ioutil.WriteFile(outputFile, testResult, 0644) //#nosec
```

Comment:
This was coming up because the master branch of architect-orb has some updated checkers and linters and go 1.16.2